### ISSUE TITLE
build on ubuntu 16.04 using libglfw3-dev package

### ISSUE THREAD

**Issue by doug-moen** [patriciogonzalezvivo/glslViewer#40]
I built glslViewer on Ubuntu 16.04. Instead of building glfw myself, I installed the package:
$ sudo apt-get install libglfw3-dev

This installed version 3.1 of the library, which is somewhat behind the latest released version.
One small change was required to make this compile:
```
--- a/src/app.cpp
+++ b/src/app.cpp
@@ -189,7 +189,7 @@ void initGL (glm::ivec4 &_viewport, bool _headless) {
         }
 
         if (_headless) {
-            glfwWindowHint(GLFW_VISIBLE, GLFW_FALSE);
+            glfwWindowHint(GLFW_VISIBLE, GL_FALSE);
         }
```
With this change, all the examples work as expected.

Would you like me to submit a pull request for this change?

**Comment by patriciogonzalezvivo** [comment]
ho! nice! Sure a PR will be great! we need to change also the README.md now that is easier : )

**Comment by doug-moen** [comment]
fixed by #42
